src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,10 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+public interface Command {
+    String execute(TaskList tasks, Ui ui, Storage storage) throws DukeException;
+}
```

Comment:
Perhaps a general header comment can be written for this execute method? That way, comments can be omitted for the execute method in the other command classes implementing this interface! 

Also, I noticed the header comments of this method in the other command classes do not have the @return line!